static/gsApp/views/onDemandBudgets/utils.tsx

```diff
@@ -5,6 +5,7 @@ import type {Organization} from 'sentry/types/organization';
 import oxfordizeArray from 'sentry/utils/oxfordizeArray';
 
 import type {
+  DataCategories,
```

Comment:
Would it be possible to use `DataCategory` instead? I'm trying to get rid of `DataCategories` in [this PR](https://github.com/getsentry/sentry/pull/90454) as it's basically a different version of `DataCategory`

Reply:
Not sure why [plural DataCategories](https://github.com/getsentry/getsentry/pull/12375/files) was added in the first place.